pages/facts.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fun Facts</title>
+    <link rel="stylesheet"  href="../styles/stylesheet.css" />
+    <link rel="stylesheet"  href="../styles/facts.css" />
+  </head>
+  <body>
+    <header>
+      <h1>FIN FACTS</h1>
+    </header>
+    <nav>
+      <a href="facts.html">Fin Facts</a>
+      <a href="index.html">Home</a>
+      <a href="gallery.html"><em>Gill</em>ary</a>
+    </nav>
+    <section class="Facts-Intro">
+    <p>Sharks. The dog of the sea, the toothy terror, the gruesome gill monster. So many titles
```

Comment:
It's a little subjective, but in this layout I might consider the header, nav and the section as part of the significant content of the page and place it inside a `main` tag.